cob_bringup/robots/cob4-1.xml

```diff
@@ -45,8 +46,8 @@
         <include file="$(find cob_bringup)/drivers/sick_flexisoft.launch">
             <arg name="robot" value="$(arg robot)"/>
         </include>
-        <!--include file="$(find cob_bringup)/tools/collision_monitor.launch">
-        </include-->
+        <include file="$(find cob_bringup)/tools/collision_monitor.launch">
```

Comment:
I'd like to keep this commented out as long as it is well tested on HW. nevertheless we should merge the rest of this PR.